style.css

```diff
@@ -857,6 +857,29 @@ html[lang="th"] textarea {
 	line-height: 1.8;
 }
 
+/* Remove letter-spacing for all non-latin alphabets */
+
+html[lang="ar"] *,
+html[lang="ary"] *,
+html[lang="azb"] *,
+html[lang="haz"] *,
+html[lang="ps"] *,
+html[lang^="zh-"] *,
+html[lang="bg-BG"] *,
+html[lang="ru-RU"] *,
+html[lang="uk"] *,
+html[lang="bn-BD"] *,
+html[lang="hi-IN"] *,
+html[lang="mr-IN"] *,
+html[lang="el"] *,
+html[lang="gu-IN"] *,
+html[lang="he-IL"] *,
+html[lang="ja"] *,
+html[lang="ko-KR"] *,
+html[lang="th"] * {
+	letter-spacing: normal !important;
```

Comment:
I have gone through style.css and gathered a list of selectors specifying letter-spacing greater than 0 (normal).

```css
h5,
.site-title,
.page .panel-content .entry-title,
.page-title,
body.page:not(.twentyseventeen-front-page) .entry-title,
.entry-meta,
.entry-footer .cat-links,
.entry-footer .tags-links,
.nav-subtitle,
.comment-metadata,
h2.widget-title,
.widget_rss .rss-date,
.widget_rss li cite,
.site-content .wp-playlist-current-item .wp-playlist-item-artist,
.twentyseventeen-customizer.twentyseventeen-front-page .twentyseventeen-panel .twentyseventeen-panel-title
```

So looking at these, this PR does reset letter-spacing with the least amount of CSS, however any plugin/child themes that need to change letter-spacing will have a difficult time. Not sure how often that is or if it is even something to worry about.

An alternative to this is to go through each of these selectors above and see if we really need to be using letter-spacing, which would make this PR unnecessary.